src/content/docs/en/guides/cms/statamic.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Headless Statamic & Astro
+description: Add content to your Astro project using Statamic as a CMS
+type: cms
+stub: false
+service: Statamic
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Statamic](https://statamic.com/) is a modern, flat-file CMS. It allows developers to easily create dynamic websites and applications while offering content editors an intuitive and user-friendly interface for managing content.
+
+## Integrating with Astro
+
+Statamic comes with a built-in [REST API](https://statamic.dev/rest-api) and [GraphQL API](https://statamic.dev/graphql) to connect your data to Astro.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+0. REST API and GraphQL API are only available in a pro version of Statamic. You can try the Pro version free on your [local machine](https://statamic.dev/tips/how-to-enable-statamic-pro#trial-mode).
+1. **An Astro project** - If you still need an Astro project, our [Installation guide](/en/install/auto/) will get you up and running quickly.
+2. **A Statamic site** - [Enable REST API](https://statamic.dev/rest-api#enable-the-api) or [GraphQL API](https://statamic.dev/graphql#enable-graphql) by adding `STATAMIC_API_ENABLED=true` or `STATAMIC_GRAPHQL_ENABLED=true` in the `.env` file and enable required resources in the API configuration file.
```

Comment:
```suggestion
1. REST API and GraphQL API are only available in a pro version of Statamic. You can try the Pro version free on your [local machine](https://statamic.dev/tips/how-to-enable-statamic-pro#trial-mode).
2. **An Astro project** - If you still need an Astro project, our [Installation guide](/en/install/auto/) will get you up and running quickly.
3. **A Statamic site** - [Enable REST API](https://statamic.dev/rest-api#enable-the-api) or [GraphQL API](https://statamic.dev/graphql#enable-graphql) by adding `STATAMIC_API_ENABLED=true` or `STATAMIC_GRAPHQL_ENABLED=true` in the `.env` file and enable required resources in the API configuration file.
```

I think this would be better to start at 1